cli/service/service.go

```diff
@@ -105,10 +106,11 @@ func Log(serviceName string, tailing bool) {
 		serviceFound := false
 		var instanceId string
 		for _, serviceInstance := range runningServices.Instances["active"] {
-			if strings.Contains(serviceInstance.SpecRef, refUrl) {
+			org, name := cutil.SplitOrgSpecUrl(refUrl)
```

Comment:
The original code allows the user to specify a substring of the service name and the command still works. We need to retain that capability.

Reply:
Done. I rewrote the condition to be more strict in the case that the org and name is inputted and only need a substring in the case that only a name is inputted.